crates/pop-contracts/src/utils/helpers.rs

```diff
@@ -28,6 +34,29 @@ pub fn parse_account(account: &str) -> Result<<DefaultConfig as Config>::Account
 		.map_err(|e| Error::AccountAddressParsing(format!("{}", e)))
 }
 
+pub fn canonicalized_path(target: &Path) -> Result<PathBuf, Error> {
+	// Canonicalize the target path to ensure consistency and resolve any symbolic links.
+	return target
+		.canonicalize()
+		// If an I/O error occurs during canonicalization, convert it into an Error enum variant.
+		.map_err(|e| Error::IO(e));
+}
+
+pub fn replace_in_file(file_path: PathBuf, replacements: HashMap<&str, &str>) -> Result<(), Error> {
```

Comment:
missing docs, pub function

Reply:
Untested

Reply:
Actually, it did have coverage: https://app.codecov.io/gh/r0gue-io/pop-cli/blob/alex%2Fcontracts-template/crates%2Fpop-contracts%2Fsrc%2Futils%2Fhelpers.rs#L45

But I agree that it's better to have a specific test for this function. I've added it.